### PR TITLE
make telegram_helper using telebot instead aiogram & update requirements

### PR DESCRIPTION
**Pull request by Andrii1000**

### CODE REVIEW

**Review by SuskyiVolodymyr — APPROVED** [review]
👌

File: borrowing/serializers.py [hunk 1]
```diff
@@ -64,6 +65,14 @@ def create(self, validated_data):
                 )
             borrowing.book.add(*books)
 
+            book_titles = ', '.join([book.title for book in books])
```
Comment (VShveda) [inline borrowing/serializers.py:68]: I think it should be double quote marks, not single quote marks

Comment (Andrii1000) [reply borrowing/serializers.py]: yes, small mistake)

**Review by VShveda — APPROVED** [review]
Good job!

**Pull request merged**